presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -527,7 +527,12 @@ private void assertPushdownSubscripts(String tableName)
     @Test
     public void testPushdownSubfields()
     {
-        assertUpdate("CREATE TABLE test_pushdown_struct_subfields(id bigint, x row(a bigint, b varchar, c double, d row(d1 bigint, d2 double)), y array(row(a bigint, b varchar, c double, d row(d1 bigint, d2 double))))");
+        assertUpdate("CREATE TABLE test_pushdown_struct_subfields(" +
```

Comment:
Would you move this refactoring into a separate commit?

Reply:
Done